src/components/Sidebar.jsx

```diff
@@ -50,6 +50,16 @@ class Sidebar extends React.Component {
           )}
           onClick={this.props.onToggleDashboard}
         />
+        <div
+          className="sidebar__plusMinus"
+          onClick={this.props.onToggleEditorTextSize}
+        >
+          {
+            this.props.textSizeIsLarge ?
+              <icon className="u__icon">&#xf010;</icon> :
+              <icon className="u__icon">&#xf00e;</icon>
```

Comment:
@pwjablonski so sorry I missed this before, but… I’m pretty sure [HTML doesn’t have an `<icon>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element)? Typically I think we would use a `<span>` for this.